### PR TITLE
Fix bug related to notifications displaying

### PR DESCRIPTION
**Pull request by duranmla**
This PR fixes the error displaying when the user has some specific notifications type awaiting by:

* Add minimal base to avoid render error on unhandled notification and show warnings
* Add extra notifications handling for common interactions and a readable default one
* Add support for notification links on all notifications

Besides:

* Static update automatic by prettier
* Minor update to README.md
* Avoid trailing commas

> Closes https://github.com/debtcollective/parent/issues/253

## Attachments

image

image

### CODE REVIEW

File: prettier.config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  trailingComma: 'all',
+  trailingComma: 'none',
```
Comment (orlando) [inline prettier.config.js]: For this, let’s just use prettier defaults

File: src/ProfileNotification.vue [hunk 2]
```diff
@@ -53,59 +53,74 @@ export default {
...
-          return getLinkForNotification(this.notification);
         case "aggregatedSystem":
           return getInboxLink();
+        default:
+          return getLinkForNotification(this.notification);
       }
     },
     summary() {
-      switch (this.notification.type) {
+      const n = this.notification;
```
Comment (orlando) [inline src/ProfileNotification.vue]: Ñame this variable notification instead of **n**

File: src/ProfileNotification.vue [hunk 3]
```diff
@@ -53,59 +53,74 @@ export default {
...
-            const { display_username: name } = this.notification.data;
+            const { display_username: name } = n.data;
             return {
               title: `${name} replied to:`,
-              content: this.notification.fancy_title
+              content: n.fancy_title
             };
           }
+        default:
+          console.warn(`Unhandled "${n.type}" notification`, n);
```
Comment (orlando) [inline src/ProfileNotification.vue]: What about integrating sentry here? We need to know when this happens to add a new case to the header, and a console warn won’t provide that feedback in production

File: prettier.config.js (hunk 1 above)
Comment (duranmla) [reply prettier.config.js]: The default is `"none"` but as there was a config file I just update it. I can just export an empty object then if you like.

File: src/services/NotificationService.js [hunk 4]
```diff
@@ -1,99 +1,75 @@
-import { getOptions, discourseEndpoint } from './service';
+import { getOptions, discourseEndpoint } from "./service";
+import { captureMessage } from "./ErrorService";
```
Comment (orlando) [inline src/services/NotificationService.js]: let's remove this unused import

**Pull request merged**